.gitignore

```diff
@@ -353,3 +353,6 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+# Chart-tgzs generated by dependency update
+charts/osm/charts
```

Comment:
this should be `charts/osm-arc/charts` as you want to ignore the dependency chart in the new osm-arc chart. please also `git rm charts/osm-arc/charts` as you currently have those files checked in.